tools/dromajo/dromajo.mk

```diff
@@ -7,7 +7,7 @@ DROMAJO_LIB_NAME = dromajo_cosim
 DROMAJO_LIB = $(DROMAJO_DIR)/lib$(DROMAJO_LIB_NAME).a
 
 # Dromajo assumes using the default bootrom
-DROMAJO_ROM = $(base_dir)/bootrom/bootrom.rv64.img
+DROMAJO_ROM = $(base_dir)/generators/testchipip/src/main/resources/testchipip/bootrom/bootrom.rv64.img
```

Comment:
This should point to the bootrom that was copied into the build directory. If you look at this: https://github.com/ucb-bar/chipyard/blob/992664220d95edf610475031fd8a82d0cd743f7f/common.mk#L98, I copy the bootrom from testchipip to the build directory.